app/src/main/java/com/kickstarter/libs/utils/ViewUtils.java

```diff
@@ -21,11 +21,22 @@
 import com.kickstarter.ui.views.AppRatingDialog;
 import com.kickstarter.ui.views.ConfirmDialog;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 import rx.functions.Action1;
 
 public final class ViewUtils {
   private ViewUtils() {}
 
+  public static Bitmap getBitmap(final @NonNull View view, final int width, final int height) {
```

Comment:
Ooh I mean this line I have commented out 
```view.layout(0, 0, width, height)```

Which would redraw the view with the specified height and width because it should already be laid out.

`view.draw(canvas)` I need because that's what takes the "screenshot" of the view.

Reply:
Ok I gotcha! I think you don't need it then being that you're already passing in the size of the view.